### DISCUSSION TITLE
Archiver

### DISCUSSION THREAD

**Discussion by adesutherland** [adesutherland/CREXX#298] [Progress Updates]
Getting near to completing f0041 (this week) - but one rabbit hole done in it, that you will like @rvjansen, is that 

```
cat member1.rxbin > library.rxbin
cat member2.rxbin >> library.rxbin
cat member3.rxbin >> library.rxbin
cat member4.rxbin >> library.rxbin 
```

concats and makes a library that will work as you once expected it to ...

**Comment by rvjansen** [comment]
That is excellent, because the simplest linkage editor is the best! And no linkage editor beats them all.
